Sql_Assignment _4_B.txt

```diff
@@ -0,0 +1,80 @@
+-- E-commerce Platform Schema
```

Comment:
Properly format queries 

   - Use consistent capitalization for keywords
   - Align JOIN conditions for better readability

Reply:
Eg; https://poorsql.com/
![image](https://github.com/user-attachments/assets/78adc0c7-d79f-46f3-8fa1-068ca0140059)